apidoc/Titanium/UI/iOS/LivePhotoView.yml

```diff
@@ -14,8 +14,19 @@ extends: Titanium.UI.View
 platforms: [iphone,ipad]
 since: "5.2.0"
 osver: {ios:{min: "9.1"}}
+excludes:
+    events: [click,dblclick,doubletap,longpress,pinch,postlayout,singletap,swipe,
```

Comment:
guess not:

- https://github.com/appcelerator/tiduck/blob/fa447ee5bd95189d2dd8c8647be4f2252a17c545/apidoc/docgen.py#L371
- https://github.com/appcelerator/tiduck/blob/fa447ee5bd95189d2dd8c8647be4f2252a17c545/apidoc/generators/jsduck_generator.py#L418